frontend/src/styles/globals.scss

```diff
@@ -35,12 +34,23 @@ body,
   // empty space at the bottom of the page, below the footer.
   // Since the user can scroll in `.non-header-wrapper`, everything outside that
   // that still overflows these elements can safely be hidden:
-  overflow: hidden;
+  // overflow: hidden;
 }
 
 html {
   scroll-behavior: smooth;
   scroll-padding-top: $layout-xl;
+  font-family: $font-stack-base;
+  height: auto;
+
+  @media #{$mq-sm} {
+    // In <Layout> all content is wrapped in a `.non-header-wrapper` that has its
+    // own `overflow` applied, to allow the header and mobile menu to remain fixed
+    // in place. However, in Blink- and WebKit-based browsers, this resulted in
+    // empty space at the bottom of the page, below the footer.
+    // Hence, a height of 100% is applied to the html document
+    height: 100%;
+  }
```

Comment:
So I simply added a media query here to only apply the `100%` height to devices sizes `>480px`, since on mobile I don't think we need to worry about content being too short.


https://user-images.githubusercontent.com/13066134/211921665-52d1da39-b1f8-440a-8a65-e58e8d9fdad6.mov

Reply:
Nice detective work! Yes, with the clear comment next to it, this seems like a fine workaround to me. Some of the [new CSS units like `dvh`](https://developer.mozilla.org/en-US/docs/Web/CSS/length#relative_length_units_based_on_viewport) might be able to provide some relief here as well, but given that I don't expect this to negatively impact maintainability, and that the result is that it works just fine for users, I don't think that's worth the time to investigate right now. Nice job!